Dockerfile

```diff
@@ -1,24 +1,25 @@
-FROM golang:1.18-alpine3.16 AS build
+FROM golang:1.23-alpine3.20 AS build
 
-ARG consul_version=1.12.3
+ARG consul_version=1.19.2
 ADD https://releases.hashicorp.com/consul/${consul_version}/consul_${consul_version}_linux_amd64.zip /usr/local/bin
-RUN cd /usr/local/bin && unzip consul_${consul_version}_linux_amd64.zip
+RUN cd /usr/local/bin && unzip -o consul_${consul_version}_linux_amd64.zip
 
-ARG vault_version=1.11.0
+ARG vault_version=1.17.5
 ADD https://releases.hashicorp.com/vault/${vault_version}/vault_${vault_version}_linux_amd64.zip /usr/local/bin
-RUN cd /usr/local/bin && unzip vault_${vault_version}_linux_amd64.zip
+RUN cd /usr/local/bin && unzip -o vault_${vault_version}_linux_amd64.zip
 
-RUN apk update && apk add --no-cache git
+RUN apk add --no-cache git libcap
 WORKDIR /src
 COPY . .
 RUN go mod tidy
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go test -trimpath -ldflags "-s -w" ./...
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -trimpath -ldflags "-s -w"
 
-FROM alpine:3.16
-RUN apk update && apk add --no-cache ca-certificates
+FROM alpine:3.20
+RUN apk add --no-cache ca-certificates
 COPY --from=build /src/fabio /usr/bin
-ADD fabio.properties /etc/fabio/fabio.properties
-EXPOSE 9998 9999
+COPY --chown=nobody:nogroup fabio.properties /etc/fabio/fabio.properties
+USER nobody:nogroup
+EXPOSE 9999 9999
```

Comment:
By default fabio listens on both 9999 and 9998, 9998 is for the management UI and 9999 is for service traffic.

Reply:
Hi, I have no idea how that happened. Good catch. Reverted.